sda/cmd/finalize/finalize.md

```diff
@@ -62,6 +62,7 @@ These settings control how `finalize` connects to the RabbitMQ message broker.
 - `BROKER_USER`: username to connect to RabbitMQ
 - `BROKER_PASSWORD`: password to connect to RabbitMQ
 - `BROKER_PREFETCHCOUNT`: Number of messages to pull from the message server at the time (default to `2`)
+- `BROKER_EXCHANGE`= the exchange name (i.e., `sda`)
```

Comment:
```suggestion
- `BROKER_EXCHANGE`: the exchange name (i.e., `sda`)
```

Follow the existing syntax

Reply:
The setting BROKER_EXCHANGE is `only` needed if the MQ is configured in that way.
Do not use the preconfigured MQ and DB as templates since they are not designed to be used in a production setting.